test/utils/initMatchers.js

```diff
@@ -1,7 +1,7 @@
 import chai from 'chai';
 import chaiDom from 'chai-dom';
 import { isInaccessible } from '@testing-library/dom';
-import { prettyDOM } from '@testing-library/react';
+import { prettyDOM } from '@testing-library/react/pure';
```

Comment:
Good thing was that this uncovered accidentally using the impure bundle. The karma and mocha runner seem to evaluate the bundles at different points in time. The node cli would not care about automatic cleanup since `beforeEach` wouldn't be defined when evaluating the module. In karma it is though which is why cleanup was called twice. This is harmless but potentially confusing.